source/includes/api/_query.md

```diff
@@ -1,204 +1,278 @@
 # Query
 
-In order to retrieve data from the datasets it is possible to send queries to the API using the SQL or Feature Service languages.
-
-Two different endpoints are provided (under the dataset path and a generic one) and the sql query can be provided via query parameters or in the body of a POST request.
+> Example of GET request for a query providing the SQL as query param:
 
 ```shell
-curl -i -H 'Authorization: Bearer your-token>' -H 'Content-Type: application/json' -XPOST 'http://api.resourcewatch.org/v1/query/<dataset_id>/' -d '{
-    "sql": "SELECT * FROM <dataset_id> limit 10"
-}
-'
+curl -i -X GET http\://api.resourcewatch.org/v1/query\?sql\=SELECT\ \*\ from\ <dataset.slug>
 ```
 
+> Example of POST request for a query providing the SQL as request body param:
+
 ```shell
-curl -i -XGET http\://api.resourcewatch.org/v1/query\?sql\=SELECT\ \*\ from\ <dataset.slug>
+curl -i -X POST 'http://api.resourcewatch.org/v1/query/<dataset.id>/' \
+-H 'Content-Type: application/json' \
+-d '{
+    "sql": "SELECT * FROM <dataset.tableName> limit 10"
+}'
 ```
 
-<aside class="notice">
-While we aim to make our query interface as broad and transparent as possible, some of the querying options described 
-below will not be available for specific dataset providers, depending on this API's implementation or limitations on the
-actual data provider's side.
-</aside>
+In order to retrieve data from datasets, you can send queries to the API using a syntax very similar to SQL. Using these endpoints, you can also download the results of a particular query.
```

Comment:
I would add a sentence here that links to the query concept docs for some background.

Reply:
Added a link to the query concept docs, thanks for pointing out :+1: